packages/aws-cdk-lib/aws-stepfunctions/lib/state-machine.ts

```diff
@@ -158,25 +157,11 @@ export interface StateMachineProps {
   readonly removalPolicy?: RemovalPolicy;
 
   /**
-   * Specifies a symmetric customer managed KMS key for server-side encryption of the state machine definition and execution history.
-   * Step Functions will reuse the key for a maximum of `kmsDataKeyReusePeriodSeconds`.
+   * The encryptionConfiguration object used for server-side encryption of the state machine definition and execution history.
```

Comment:
Using the property name as the description doesn't provide any information. Perhaps:

```suggestion
   * Configures server-side encryption of the state machine definition and execution history.
```